linera-service/tests/end_to_end_tests.rs

```diff
@@ -1509,6 +1513,96 @@ async fn test_reconfiguration(network: Network) {
     }
 }
 
+#[test_log::test(tokio::test)]
+#[ignore] // TODO(#901)
```

Comment:
nit: at the beginning of a line `// TODO(#901): Node service should track newly opened chains`